README.md

```diff
@@ -78,6 +78,30 @@ Follow the given instructions for Login into the app.
 
 To run the tests run: `python manage.py test`.
 
+### Run with Docker
+
+1. Make sure the latest version of docker and docker-compose are installed.
+
+- For **docker** run ``docker -v`` from your terminal. If docker is not installed, install it from [here](https://docs.docker.com/engine/install/).
+- For **docker-compose** run ``docker-compose -v`` from terminal. If it is not installed then install it by running ``pip install docker-compose`` or from [here](https://docs.docker.com/compose/install/).
+
+2. Change Database HOST in ``settings.py``
+
+- Go to ``main/settings.py``
+- Under databases change host to db.
+
+3. **Build docker image**
+
+Execute ``sudo docker-compose run web`` to build the image.
+
+4. **Run docker image**
+
+Run ``sudo docker-compose up`` from the root directory of project. Navigate to `http://localhost:8000` in your browser.
+
+**Note** 
+- Run `docker-compose up --build` to rebuild the image.
+- To **interact** with docker containers use ``docker exec -it {container id} bash`` from your terminal. Container id can be found using ``docker ps``.
```

Comment:
@codesankalp Added this command if someone wants to run commands like `python manage.py runserver`.

Reply:
Thanks @devkapilbansal , for adding this.